renderer.js

```diff
@@ -121,11 +133,15 @@ function enterFullscreen () {
 
 function exitFullscreen () {
   if (isFullscreen()) return;
-  fixUpCanvas();
-  resetCamera(camera);
+  handleResize();
   safeToRenderStereo = false;
 };
 
+function handleResize() {
+  fixUpCanvas();
```

Comment:
it looks like you have some of the changes from master in here. that's probably happened because you're working in `master`. if you work in a branch and pull the latest changes, things should be fine:

```bash
git checkout master
git checkout -b improvements
git checkout master
git reset --hard 21c22f2
git pull --rebase
git checkout improvements
git rebase master
git push annacruz head -f
```

let me know if that works okay for you :smile: